src/DataFixtures/UserFixtures.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\DataFixtures;
+
+use App\Entity\User;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Common\Persistence\ObjectManager;
+
+
+class UserFixtures extends Fixture
+{
+    public function load(ObjectManager $manager)
+    {
+        $user = new User();
+        $user->setUsername('John')->setEmail("John123@gmail.com")->setRoles(array('ROLE_USER'))->setRegisterDate(new \DateTime())->setEnabled(1)->setPlainPassword("JohnPassword");
```

Comment:
You can remove the setRoles and setEnabled if you want, because the values your are setting here are the same as the default values
```php
    private $roles = ["ROLE_USER"];
    private $enabled = true;

```